cirq-core/cirq/ops/raw_types_test.py

```diff
@@ -662,6 +662,7 @@ def test_tagged_operation_forwards_protocols():
     assert cirq.commutes(tagged_x, clifford_x)
     assert cirq.commutes(clifford_x, tagged_x)
     assert cirq.commutes(tagged_x, tagged_x)
+    assert cirq.phase_by(tagged_x, 0.125, 1, default=None) is None
```

Comment:
Nit - we can use a valid qubit_index and test both tagged and untagged operations behave the same in phase_by:
```suggestion
    assert cirq.phase_by(clifford_x, 0.125, 0, default=None) is None
    assert cirq.phase_by(tagged_x, 0.125, 0, default=None) is None
```